concent_api/core/payments/bankster.py

```diff
@@ -391,3 +391,8 @@ def discard_claim(deposit_claim: DepositClaim) -> bool:
             claim_removed = True
 
     return claim_removed
+
+
+def remove_deposit_claim_if_transaction_successed_in_blockchain(deposit_claim: DepositClaim) -> None:
+    if DepositClaim.objects.filter(pk=deposit_claim.pk).exists():
```

Comment:
is there a possibility that this `deposit_claim` doesn't exist at this point?

Reply:
BTW:
"successed" -> "succeeded"

Reply:
good idea :+1: . It is checked by function calling, so there is no need to check it again